packages/utils/src/graphql/entities.ts

```diff
@@ -80,10 +80,11 @@ export function getAllEntitiesRelations(_schema: GraphQLSchema | string | null):
   const modelRelations = {models: [], relations: [], enums: [...enums.values()]} as GraphQLModelsRelationsEnums;
   const derivedFrom = schema.getDirective('derivedFrom');
   const indexDirective = schema.getDirective('index');
+  assert(derivedFrom && indexDirective, 'derivedFrom and index directives are required');
   for (const entity of entities) {
     const newModel: GraphQLModelsType = {
       name: entity.name,
-      description: entity.description,
+      description: entity.description || undefined,
```

Comment:
if description is "" or 0 it will be converted to undefined, its better to use `??`. This applies to the other cases as well

```suggestion
      description: entity.description ?? undefined,
```

Reply:
fix